src/qt/rpcconsole.cpp

```diff
@@ -578,7 +576,7 @@ RPCConsole::RPCConsole(interfaces::Node& node, const PlatformStyle *_platformSty
     // based timer interface
     m_node.rpcSetTimerInterfaceIfUnset(rpcTimerInterface);
 
-    setTrafficGraphRange(INITIAL_TRAFFIC_GRAPH_MINS);
```

Comment:
Why are we losing this default constant?

Reply:
because it isn't needed - on startup it "defaults" to the first non-full range